CHANGELOG.md

```diff
@@ -14,7 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 - `py.typed` file to enable the use of type checkers on the user side
-- `GaussianSurrogate` base class for surrogate models with Gaussian posteriors
+- `IndependentGaussianSurrogate` base class for surrogate models providing independent 
```

Comment:
(Putting this here such that it is a proper comment)

It seems like the `docs` pipeline is failing for a good reason this time: When just executing the failed example (`surrogate_params`) normally via `python`, I get the following error:
<img width="794" alt="image" src="https://github.com/user-attachments/assets/cfc696a8-fa1f-4d68-b973-f665ee337617">

Reply:
Seems you were already in the middle of the review while I fixed it 🙃